modules/mobile/app/workers/mobile/v0/vet360_linking_job.rb

```diff
@@ -15,16 +28,13 @@ def perform(uuid)
         user = IAMUser.find(uuid) || User.find(uuid)
         raise MissingUserError, uuid unless user
 
-        result = Mobile::V0::Profile::SyncUpdateService.new(user).await_vet360_account_link
-        Rails.logger.info('Mobile Vet360 account linking succeeded for user with uuid',
-                          { user_uuid: uuid, transaction_id: result.transaction_id })
+        result = VAProfile::Person::Service.new(user).init_vet360_id
+        Rails.logger.info('Mobile Vet360 account linking request succeeded for user with uuid',
+                          { user_uuid: uuid, transaction_id: result.transaction.id })
```

Comment:
I don't think there's any value in this transaction id. One thing that may not be very clear from the diff is that this transaction is different from the transaction that was previously being returned. Neither of them truly valuable unless we want to digging deeply into logs with the VAProfile team.